public/app/features/alerting/unified/rule-list/FilterView.tsx

```diff
@@ -127,7 +127,14 @@ function FilterViewResults({ filterState }: FilterViewProps) {
             case 'datasource':
               return <DataSourceRuleLoader key={key} rule={rule} groupIdentifier={groupIdentifier} />;
             default:
-              return <UnknownRuleListItem key={key} rule={rule} groupIdentifier={groupIdentifier} />;
+              return (
+                <UnknownRuleListItem
+                  key={key}
+                  ruleName="Unknown rule"
```

Comment:
maybe `unknown rule type` ?

Reply:
Ideally this should be translated as well